oeps/processes/oep-0060-proc-sec-group.rst

```diff
@@ -156,6 +208,15 @@ What do I do if I am an operator and someone reports a vulnerability to me?
 * If you believe this is a vulnerability in the Open edX Platform, please forward the issue to security@openedx.org
   or ask the reporter to re-file the issue to security@openedx.org
 
+What will happen if a report is accidentally sent to security@openedx.org for the operation of my Open edX instance?
+
+* Please let security@openedx.org know the best email to forward such reports to, along with your Open edX instance name, domain, and separate contact information for an indidual responsible for security at your organization. The security working group will do its best to forward such reports to the correct organization.
+
+How do I receive notification of the release of upcoming security patches?
+
+* Please watch the Open edX Discourse Security Announcements topic at https://discuss.openedx.org/c/announcements/security/19. If you are logged in, the button with a bell icon on the top right corner above the topic list will let you customize how you are notified of security announcements.
+
+
 Change History
```

Comment:
Don't forget to add a section to the change history.

Reply:
Done.